packages/back-end/src/controllers/reports.ts

```diff
@@ -34,10 +36,7 @@ export async function postReportFromSnapshot(
     throw new Error("Invalid snapshot id");
   }
 
-  const experiment = await ExperimentModel.findOne({
-    organization: org.id,
-    id: snapshot.experiment,
-  });
+  const experiment = await getExperimentById(snapshot.experiment);
```

Comment:
Experiment ids are not unique across organizations.  So the org id always has to be part of the query